src/stats.rs

```diff
@@ -32,7 +31,7 @@ pub struct Stats {
     pub tunnel_connections: usize,
 
     /// Messages sent by us on different routes.
-    routes: [usize; MIN_GROUP_SIZE],
+    routes: Vec<usize>,
```

Comment:
It is already: just pass a different number to `Stats::new(num_routes)`. Unless you want it to be compile-time constant still?